src/test/java/com/endava/internship/mocking/service/PaymentServiceTest.java

```diff
@@ -1,31 +1,118 @@
 package com.endava.internship.mocking.service;
 
+import com.endava.internship.mocking.model.Payment;
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import com.endava.internship.mocking.repository.PaymentRepository;
+import com.endava.internship.mocking.repository.UserRepository;
+import com.endava.internship.mocking.service.PaymentService;
+import com.endava.internship.mocking.service.ValidationService;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
 
+import java.util.Optional;
+import java.util.UUID;
+import java.util.List;
+import java.util.Arrays;
+import java.util.ArrayList;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyInt;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
 @ExtendWith(MockitoExtension.class)
 class PaymentServiceTest {
 
+    @Mock
+    private PaymentRepository paymentRepository;
 
-    @BeforeEach
-    void setUp() {
+    @Mock
+    private UserRepository userRepository;
 
-    }
+    @Mock
+    private ValidationService validationService;
+
+    @InjectMocks
+    private PaymentService paymentService;
+
+    @Captor
+    private ArgumentCaptor<Payment> captor;
+
+    private final static Double amount = 100.0;
+
+    private final static Integer id = 1;
+
+    private final static User user = new User(id, "Dmitry", Status.ACTIVE);
+
+    private final static Payment payment = new Payment(user.getId(), amount, "Payment from user " + user.getName());
+
+    private final static String message = "New Message";
+
+    private final static UUID uuid = UUID.randomUUID();
+
+    @BeforeEach
+    void setUp() {}
 
     @Test
     void createPayment() {
+        when(userRepository.findById(anyInt())).thenReturn(Optional.of(user));
```

Comment:
please avoid using anyInt(), any() etc. If it is possible use concrete values

Reply:
changed